packages/core-api/src/app/configLoaders/windowConfigLoader.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { AppConfigLoader } from '../types';
+import { AppConfig } from '@backstage/config';
+
+declare global {
+  interface Window {
+    __APP_CONFIG__: AppConfig;
```

Comment:
Since most other mechanisms for setting config have an APP_CONIFG prefix I thought I'd keep it the same. Do we need the double underscores? I suppose users shouldn't access it directly so it's an extra "warning"